### PR TITLE
Don't mark newly founded groups probably inactive

### PR DESCRIPTION
**Pull request by falusi94**

### CODE REVIEW

**Comment by falusi94** [comment]
[Új körnél is kiírjuk, hogy 'A kör valószínűleg inaktív!'](https://trello.com/c/yHdYlOoq/291-%C3%BAj-k%C3%B6rn%C3%A9l-is-ki%C3%ADrjuk-hogy-a-k%C3%B6r-val%C3%B3sz%C3%ADn%C5%B1leg-inakt%C3%ADv)

File: test/models/group_test.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    Timecop.travel(time) do
+      assert group.inactive?
+    end
+  end
+
+  test 'group with evaluation in last semester is active' do
+    group = groups(:babhamozo)
+    evaluation = evaluations(:babhamozo_2018)
+
+    SystemAttribute.stubs(:semester).returns(Semester.new(evaluation.semester).next!)
```
Comment (emanreyalp) [inline test/models/group_test.rb]: Nice ;)

**Comment by emanreyalp** [comment]
I found this article about the `exists?` and `any?`
https://semaphoreci.com/blog/2017/03/14/faster-rails-how-to-check-if-a-record-exists.html
Interesting that if the data are eagerloaded, than the `any?` could be faster. Because we don't hit the database in that case.

**Comment by falusi94** [comment]
> I found this article about the `exists?` and `any?`
> https://semaphoreci.com/blog/2017/03/14/faster-rails-how-to-check-if-a-record-exists.html
> Interesting that if the data are eager loaded, than the `any?` could be faster. Because we don't hit the database in that case.

Yes, but it makes sense. I will change then, because we use it on group index and show pages where the evaluations are not loaded.

File: test/models/group_test.rb (hunk 1 above)
Comment (falusi94) [reply test/models/group_test.rb]: You can learn things in Madrid 🌴

I've also checked RSpec's mock library and the minitest's built-in too, but using the first one is bit painful outside of RSpec and don't talk about the second one.

**Pull request merged**